src/seedu/addressbook/Main.java

```diff
@@ -87,7 +85,7 @@ private void runCommandLoopUntilExitCommand() {
             CommandResult result = executeCommand(command);
             recordResult(result);
             ui.showResultToUser(result);
-
+            showListAfterAddOrDelete(userCommandText);
```

Comment:
Perhaps this is not the right place to have this operation? SLAP could be increased if this operation is taken care of in `.showResultToUser()`.